modules/svg/svg.go

```diff
@@ -4,10 +4,41 @@
 
 package svg
 
-// SVGs contains discovered SVGs
-var SVGs map[string]string
+import (
+	"fmt"
+	"html/template"
+	"regexp"
+	"strings"
+
+	"code.gitea.io/gitea/modules/html"
+)
+
+var (
+	// SVGs contains discovered SVGs
+	SVGs map[string]string
+
+	widthRe  = regexp.MustCompile(`width="[0-9]+?"`)
+	heightRe = regexp.MustCompile(`height="[0-9]+?"`)
+)
 
 // Init discovers SVGs and populates the `SVGs` variable
 func Init() {
 	SVGs = Discover()
 }
+
+// Render render icons - arguments icon name (string), size (int), class (string)
+func RenderHTML(icon string, others ...interface{}) template.HTML {
+	size, class := html.ParseSizeAndClass(16, "", others...)
+
+	if svgStr, ok := SVGs[icon]; ok {
+		if size != 16 {
```

Comment:
Should we perhaps extract that into a `const defaultSize = 16`?

Reply:
done.